src/lib/ensureSSL.ts

```diff
@@ -0,0 +1,17 @@
+/*
+* Makes sure that any http requests get redirected to https
+*/
+
+import { parse } from "url"
+const { APP_URL } = process.env
+
+export const ensureSSL = (req, res, next) => {
+  const protocol = req.protocol
+  const isSecure = req.secure || protocol === "https:"
```

Comment:
Can you give an example of when `req.secure` would be false but `protocol === 'https:'` would be true? (This is most likely a question for my own education than anything else.)

Reply:
@pepopowitz in theory, just checking against protocol should be fine. Originally I only had `req.secure` which returns true if the protocol is `https:` but I added `req.protocol` because I thought that made it a little more clear of what was happening in the test.

I can remove the `req.secure` check.

Reply:
No problem, let's leave it as-is!

Reply:
This may be the source of the problem - in my experience `req.protocol` evaluates to `http` or `https`, but here with `"https:"` and including the `:` delimiter (which is usually parsed out) this would never evaluate to true